local_settings.py.example

```diff
@@ -1,69 +0,0 @@
-# Setting Overrides
```

Comment:
Is this being removed because we're laying it down with Salt?

Reply:
yes